operator/builtin/input/journald/journald.go

```diff
@@ -184,7 +186,7 @@ func (operator *JournaldInput) poll(ctx context.Context) error {
 
 	err = cmd.Start()
 	if err != nil {
-		return fmt.Errorf("start journalctl: %s", err)
+		return fmt.Errorf("start journalctl with command '%s': %s", cmd.String(), err)
```

Comment:
Likewise, you don't need to call `.String()`, as `fmt` will do so if the struct implements it.

Reply:
Right, good catch.